app/assets/javascripts/controllers/catalog/catalog_item_form_controller.js

```diff
@@ -403,5 +403,41 @@ ManageIQ.angular.app.controller('catalogItemFormController', ['$scope', 'catalog
     });
   }
 
+  $scope.cancelCopyProvisioning = function() {
+    closeConfirmationModal();
+  }
+
+  $scope.copyProvisioning = function() {
+    closeConfirmationModal();
+    vm.formOptions();
+    vm.catalogItemModel.retirement_repository_id = vm.catalogItemModel.provisioning_repository_id;
+    vm.catalogItemModel.retirement_playbook_id = vm.catalogItemModel.provisioning_playbook_id;
+    vm.catalogItemModel.retirement_machine_credential_id = vm.catalogItemModel.provisioning_machine_credential_id;
+    vm.catalogItemModel.retirement_network_credential_id = vm.catalogItemModel.provisioning_network_credential_id;
+    vm.catalogItemModel.retirement_cloud_credential_id = vm.catalogItemModel.provisioning_cloud_credential_id;
+    vm.catalogItemModel.retirement_inventory = vm.catalogItemModel.provisioning_inventory;
+    vm.catalogItemModel.retirement_dialog_existing = vm.catalogItemModel.provisioning_dialog_existing;
+    vm.catalogItemModel.retirement_dialog_id = vm.catalogItemModel.provisioning_dialog_id;
+    vm.catalogItemModel.retirement_dialog_name = vm.catalogItemModel.provisioning_dialog_name;
+    vm.catalogItemModel.retirement_key = '';
+    vm.catalogItemModel.retirement_value = '';
+    vm.catalogItemModel.retirement_variables = angular.copy(vm.catalogItemModel.provisioning_variables);
```

Comment:
Digging into the UI a bit more - I'm asking about the values in the 'Remove Resources' dropdown in the Retirement section.  Looking at the code - and the UI - it seems that the user would still be required to pick what option they would want after copying over the Provision Tab information, and/or it would default to 'Yes'. - I'm asking if that is okay in all cases when copying over from the Provision tab.  @gmcculloug - any thoughts?